smoke/helpers.go

```diff
@@ -9,13 +10,34 @@ import (
 	. "github.com/onsi/gomega/gexec"
 )
 
+var DEFAULT_TIMEOUT = 30 * time.Second
+
+func SetBackend(appName string) {
+	config := GetConfig()
+	if config.Backend == "diego" {
+		EnableDiego(appName)
+	} else if config.Backend == "dea" {
+		DisableDiego(appName)
+	}
+}
+
 func EnableDiego(appName string) {
+	guid := GetAppGuid(appName)
+	Eventually(cf.Cf("curl", "/v2/apps/"+guid, "-X", "PUT", "-d", `{"diego": true}`), DEFAULT_TIMEOUT).Should(Exit(0))
+}
+
+func DisableDiego(appName string) {
```

Comment:
You are right DisableDiego is not used outside of the package. Question is if we should keep it like it is to be consistent to EnableDiego (which is used in windows test case)?